src/main/java/org/openrewrite/kotlin/KotlinTypeGoat.kt

```diff
@@ -19,12 +19,16 @@ package org.openrewrite.kotlin
 
 import java.lang.Object
 
+// Whenever this class is changed, make a corresponding change in KotlinTypeGoat in the main resources folder.
```

Comment:
FYI I added a comment that was missing from JavaTypeGoat to ensure the two files are updated together.